lib/fixedpointmath/tests/test_fp_math.py

```
@@ -18,6 +18,35 @@ class TestFixedPointMath(unittest.TestCase):
     NEG_INF = FixedPoint("-inf")
     NAN = FixedPoint("nan")
 
+    def test_clip(self):
+        """Test clip method with finite values"""
+        assert FixedPointMath.clip(0, 1, 2) == 1
+        assert FixedPointMath.clip(-1, -5, 5) == -1
```

this should throw an error, min is > max